docs/source/terms_and_model.rst

```diff
@@ -346,7 +346,119 @@ Location for Variation.
   location is non-specific (e.g., a range) or symbolic (a gene).
 
 
+.. _chromosome_location:
+
+ChromosomeLocation
+$$$$$$$$$$$$$$$$$$$$
+
+**Biological definition**
+
+Imprecise chromosomal locations based on named landmarks assayed
+through chromosomal staining, FISH, recombination studies, or other
+low-resolution methods.
```

Comment:
This is fine, though I might have left off the notion of precision / resolution. While correct, it feels superfluous.

Reply:
+1 @ahwagner . This also could lead to confusion since different band resolutions exist w/o good standardization (though the ISCN provides references):

    Chromosomal locations based on named landmarks assayed through chromosomal staining, FISH, recombination studies, or other methods referencing banded karyograms.

... is probably better suited.

Reply:
+1 @mbaudis

Reply:
Okay!

Changed to "Imprecise chromosomal locations based on named landmarks."

And, sorry @mbaudis... I hadn't seen your comments until just now. Thanks for chiming in.